src/jit/gentree.cpp

```diff
@@ -7486,6 +7418,133 @@ GenTree* Compiler::gtCloneExpr(
     return copy;
 }
 
+//------------------------------------------------------------------------
+// gtCloneExprCallHelper: clone a call tree
+//
+// Notes:
+//    Do not invoke this method directly, instead call either gtCloneExpr
+//    or gtCloneCandidateCall, as appropriate.
+//
+//    See notes on gtCloneExpr for use other arguments.
```

Comment:
What does "use other arguments" mean?

Reply:
I was trying to leave a forwarding pointer to the parent method comments for those args instead of just duplicating the comments here, but didn't quite succeed. Will try and make it clearer.

Reply:
Seems like the args part documents them well enough, so will just remove this bit of confusing text.